### PR TITLE
Add CLU

### PR DESCRIPTION
**Pull request by Bro-mine**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

**Comment by Bro-mine** [comment]
Not sure if I had to update the readme or not, apologies if i messed it up, it is my first time contributing to a public repo

**Review by MrBrain295 — APPROVED** [review]
Looks good 👍

**Comment by MrBrain295** [comment]
F.Y.I you do have to add it to the readme

**Comment by Bro-mine** [comment]
> F.Y.I you do have to add it to the readme

I did, eventually

File: c/CLU.clu [hunk 1]
```diff
@@ -0,0 +1,4 @@
+start_up = proc ()
+ po: stream := stream$primary_output ()
+ stream$putl (po, "Hello World")
```
Comment (Richienb) [inline c/CLU.clu:3]: Should these lines have a space of indent?

Comment (Bro-mine) [reply c/CLU.clu:3]: Lines 2 and 3 should have indentation, not necessarily only one space, sorry for that one.

**Pull request merged**